store/cache.go

```diff
@@ -100,13 +102,14 @@ func (cm *CacheManager) GetPath(cacheKey string) string {
 }
 
 // Get gets the given cacheKey content and puts it into targetPath
-func (cm *CacheManager) Get(cacheKey, targetPath string) error {
+func (cm *CacheManager) Get(cacheKey, targetPath string) bool {
 	if err := os.Link(cm.path(cacheKey), targetPath); err != nil {
-		return err
+		return errors.Is(err, os.ErrExist)
```

Comment:
In the case of cache hit where the target file already exists:
  - Not logging cache hit  -> Seems inconsistent with store_download.go?
  - Not updating timestamp -> Is the intention to retain recent additions in favour of high hitters?

Reply:
Thanks. I updated the condition to be more consistent